packages/validator-bonds-sdk/src/web3.js/stakeAccount.ts

```diff
@@ -210,9 +210,8 @@ export async function loadStakeAccounts({
   addresses = addresses
     .map(d => (isWithPublicKey(d) ? d.publicKey : d))
     .map(d => d as PublicKey)
-  const accounts = (
-    await getMultipleAccounts({ connection: innerConnection, addresses })
-  )
+  const accountsFirst = await getMultipleAccounts({ connection, addresses })
+  const accounts = accountsFirst
```

Comment:
```suggestion
  const accounts = (
    await getMultipleAccounts({ connection: innerConnection, addresses })
  )
```
(non-blocking)
minor suggestion - the helper var `accountsFirst` is no longer needed after removing the debug logs